packages/sit-onyx/src/components/OnyxCheckboxGroup/OnyxCheckboxGroup.vue

```diff
@@ -52,9 +52,13 @@ const checkAllLabel = computed(() => {
 </script>
 
 <template>
-  <fieldset :class="['onyx-checkbox-group', densityClass]" :disabled="props.disabled">
-    <legend v-if="props.headline" class="onyx-checkbox-group__label">
-      <OnyxHeadline is="h3">{{ props.headline }}</OnyxHeadline>
+  <fieldset
+    :class="['onyx-checkbox-group', densityClass]"
+    :disabled="props.disabled"
+    :aria-label="props.label"
+  >
+    <legend v-if="!props.hideLabel" class="onyx-checkbox-group__label">
+      <OnyxHeadline is="h3">{{ props.label }}</OnyxHeadline>
```

Comment:
in case this is visible, shouldn't aria-label be udnefined and instead e.g. aria-described-by can be used pointing to the legend?
I have the feeling that else, a screen reader might read out the label twice.

Reply:
I checked this with Jonathan, he said thats fine :)